staging/src/k8s.io/metrics/pkg/client/custom_metrics/multi_client.go

```diff
@@ -46,7 +46,7 @@ func PeriodicallyInvalidate(cache AvailableAPIsGetter, interval time.Duration, s
 		case <-ticker.C:
 			cache.Invalidate()
 		case <-stopCh:
-			break
+			return
```

Comment:
return seems okay as well!

I can see a bug if someone happens to put something after the for loop, but I think it's a low risk.